__tests__/api/app.test.js

```diff
@@ -176,4 +176,70 @@ describe('Testing app', () => {
       });
     });
   });
+
+  describe('Endpoint /api/articles/:article_id/comments', () => {
+    describe('Method GET', () => {
+      test('GET: 200 status', () => {
+        return request(app).get('/api/articles/1/comments').expect(200);
+      });
+
+      test('GET: 200 status responds with an array of comments for the given article_id of which each comment', () => {
+        return request(app)
+          .get('/api/articles/3/comments')
+          .then(({ body }) => {
+            const { comments } = body;
+            const expectComments = [
+              {
+                comment_id: 11,
+                votes: 0,
+                created_at: '2020-09-19T23:10:00.000Z',
+                author: 'icellusedkars',
+                body: 'Ambidextrous marsupial',
+                article_id: 3,
+              },
+              {
+                comment_id: 10,
+                votes: 0,
+                created_at: '2020-06-20T07:24:00.000Z',
+                author: 'icellusedkars',
+                body: 'git push origin master',
+                article_id: 3,
+              },
+            ];
+
+            expect(comments).toHaveLength(2);
+            expect(comments).toMatchObject(expectComments);
+          });
+      });
```

Comment:
Please add another test (200) where the article has no comments. This should respond with an empty array.

Reply:
Done